test/common/testinfra.py

```diff
@@ -745,6 +745,11 @@ def update_known_issue(self, number, err, details, context, timestamp=None):
 """.format(err_key, link, latest_occurrences))
                     updated = True
 
+                # This comment is already too long
+                body = "<hr>".join(parts)
+                if len(body) >= 65536:
+                    break
+
                 # update comment, no need to check others
                 return self.patch("issues/comments/{0}".format(comment['id']), { "body": "<hr>".join(parts) })
```

Comment:
We can reuse the body variable here or not introduce it above. Storing it in a variable for single time use and then not using it seems inconsistent.